jexl-eval/Cargo.toml

```diff
@@ -8,8 +8,8 @@ license = "MPL-2.0"
 repository = "https://github.com/mozilla/jexl-rs"
 
 [dependencies]
-# jexl-parser = {path = "../jexl-parser"}
-jexl-parser = "0.1.1"
+jexl-parser = { version = "0.1.0", path = "../jexl-parser" }
```

Comment:
I see a few crates in the wild do this, and they seem OK without republishing. Ref: https://github.com/tokio-rs/tokio/blob/8feebab7cdef2fbeb810d18509b1443b6b9f60b1/tokio/Cargo.toml#L95

Reply:
Yesss! that's what we needed!